rpcs3/Emu/Cell/Modules/cellMic.h

```diff
@@ -323,6 +323,7 @@ class microphone_device
 private:
 	template <u32 bytesize>
 	static inline void variable_byteswap(const void* src, void* dst);
+	inline u32 convert_16_bit_pcm_to_float(const std::vector<u8>& buffer, usz num_bytes);
```

Comment:
The return type has to be usz to prevent a warning on windows
```suggestion
	inline usz convert_16_bit_pcm_to_float(const std::vector<u8>& buffer, usz num_bytes);
```

Reply:
alternatively you can cast the return of the function to:
`return static_cast<u32>(num_bytes * 2);`
Or you will have a different warning when passing bufsize_float to write_bytes...